src/main/java/loqor/ait/tardis/data/landing/LandingPadManager.java

```diff
@@ -58,7 +57,7 @@ private LandingPadRegion claim(ChunkPos pos, int y) {
         LandingPadRegion created = new LandingPadRegion(pos, y, "");
         chunk.setAttached(PERSISTENT, created);
 
-        Network.syncTracked(Network.Action.ADD, this.world, pos);
+        LandingPadManager.Network.syncTracked(LandingPadManager.Network.Action.ADD, this.world, pos);
```

Comment:
Thats another reason why i think NetworkUtil should stay as NetworkUtil
Less confusion